packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -119,12 +119,12 @@ export class AppComponent implements OnInit {
   }
 
   login() {
-    this.oktaAuth.loginRedirect('/profile');
+    this.oktaAuth.signIn('/profile');
```

Comment:
Should be:

```
this.oktaAuth.signInWithRedirect({
  originalUri: '/profile'
})
```

There's a [migrating guide](https://github.com/okta/okta-angular/blob/master/MIGRATING.md#loginredirect-is-replaced-by-signinwithredirect) that can help.

Reply:
@aarongranick-okta 
Done!